boards/arm/sensortile_box_pro/board.c

```diff
@@ -11,7 +11,8 @@
 #include <zephyr/usb/usb_device.h>
 #include <zephyr/drivers/uart.h>
 
-#ifdef CONFIG_USB_DEVICE_STACK
+#if defined(CONFIG_CONSOLE) && defined(CONFIG_UART_CONSOLE)
```

Comment:
I'm not sure to understand the dependency with `CONFIG_UART_CONSOLE` here.
Isn't `CONFIG_CONSOLE` sufficient ?

Reply:
@erwango, in my understanding, `zephyr_cdc_acm_uart` is a USB implementation for UART console.
I could imagine situation where user wants to have a console, but not over UART, eg. (RAM_CONSOLE), in this case, I would not want to enable the `zephyr_cdc_acm_uart`.

Reply:
> in my understanding, zephyr_cdc_acm_uart is a USB implementation for UART console.

Exact! I forgot about it. It is then perfectly fine indeed.